gems/quilt_rails/README.md

```diff
@@ -215,18 +217,205 @@ class ReactController < ApplicationController
 end
 ```
 
+### Performance
+
+#### Reportable
+
+The `Quilt::Performance::Reportable` mixin is intended to be used in Rails controllers, and provides only the `process_report` method. This method handles parsing an incoming report from [@shopify/react-performance's](https://www.npmjs.com/package/@shopify/react-performance) `<PerformanceReport />` component (or a custom report in the same format) and sending it to your application's StatsD endpoint as `distribution`s using [`StatsD-Instrument`](https://rubygems.org/gems/statsd-instrument).
+
+> **Note** `Quilt::Performance::Reportable` is usable even if you are using purely client-side React through `sewing_kit_script_tag`.
```

Comment:
I think this would be more clear if you say you **don't** need to be using any of the react-server stuff. It reads a little weird to me right now